waspc/data/Generator/templates/sdk/wasp/server/auth/hooks.ts

```diff
@@ -77,6 +82,21 @@ type OnAfterSignupHookParams = {
   req: ExpressRequest
 } & InternalAuthHookParams
 
+type OnAfterEmailVerifiedHookParams = {
+  /**
+   * The email address that was verified.
+  */
+  email: string
+  /**
+   * The user who verified their email.
+  */
+  user: FindAuthWithUserResult['user']
+  /**
+   * Request object that can be used to access the incoming request.
```

Comment:
Let's maybe add:
```suggestion
   * Request object that can be used to access the user's incoming verification request.
```
And update the rest of the hooks to explain whose requests we're dealing with. The current documentation comments don't say much (of course `req` is the incoming request). I know it's a little out of scope, but seems small enough. Tell me if you disagree :)